fem/eltrans.cpp

```diff
@@ -458,6 +460,8 @@ int IsoparametricTransformation::OrderW() const
          return (FElem->GetOrder() - 1) * FElem->GetDim();
       case FunctionSpace::Qk:
          return (FElem->GetOrder() * FElem->GetDim() - 1);
+      case FunctionSpace::Uk:
+         return (FElem->GetOrder() * FElem->GetDim());
```

Comment:
I think this can actually return the same value as `Qk`.

Reply:
I had tried that initially but some integrals were causing trouble. I had assumed this was due to the different ways the Jacobian enters some of the integrals involving H(Curl) and H(Div) bases. I just reran the unit tests with the lower `OrderW` and they all pass. Those issues must have been resolved by a bug fix.

I'll lower this as you suggest. Thanks!